client/my-sites/email/email-providers-in-depth-comparison/types.ts

```diff
@@ -1,11 +1,15 @@
+import { IntervalLength } from 'calypso/my-sites/email/email-providers-stacked-comparison/provider-cards/utils';
```

Comment:
Should this not import the type only?

Reply:
I'm not sure to understand, would you mind clarifying? First `IntervalLength` is an enum and not a type. We also don't import anything else on this line.

Reply:
I didn't realise this was an enum and not a type -- I just noticed it being used as a type in the code. No need to change!